src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -93,6 +93,13 @@
     <!--     
       Exclude dependencies that are not inserted from Roslyn repo.
       Note: can't use globs due to https://github.com/Microsoft/msbuild/issues/3791.
+      
+      these dependency should be removed once StreamJsonRpc removes support for
+      net472 - https://github.com/dotnet/roslyn/issues/34558
```

Comment:
Not "removes support for" - - _adds_ a build target for.

Reply:
alright. I can update the comment.

Reply:
BTW, I'm spinning up a 2.0 release right now, so within a couple hours, you'll have the net472 target version you need.

Reply:
Oh, never mind. I just re-discovered why I hadn't done this yet, and it won't be done today. :(

Reply:
I think we would also need to have that one inserted into VS - IIRC the net46 is currently inserted. Is that what you've re-discovered?

Reply:
I don't think you'll _need_ it inserted into VS before making all your changes here. The types are equivalent and VS has access to System.ValueTuple.dll so loading that is no problem. That said, yes I agree it makes sense to follow up the availability of a net472 build with an insertion that updates VS to consume it.

What I discovered was that our v2.0 branch is currently building a -beta package because to fix a bug we took an update on another package update (Nerdbank.Streams) that isn't yet in a stable release. So I have to release a stable nerdbank.streams 2.1 version first, then we can release a stable streamjsonrpc 2.0 update with that bug fixed along with the net472 build.

Reply:
So the new streamjsonrpc (2.0.167) is on nuget.org now. But I have to insert it into VS since it also needs a new Nerdbank.Streams assembly which requires some more complicated insertion steps. I've included you two on the PR.

Reply:
The 2.0.167 insertion to VS is complete just now:

[Pull Request 172386](https://dev.azure.com/devdiv/DevDiv/_git/VS/pullrequest/172386?_a=overview): Insert StreamJsonRpc 2.0.167